### ISSUE TITLE
review our backup procedures

### ISSUE THREAD

**Issue by JosephMcArthur** [oaworks/discussion#1654]
> Lastly, we need to review our backup procedures as they have not been checked for a long time. This is another half day to a day, if we go so far as ensuring we can recreate from backup, which we should do.

From: https://github.com/OAButton/discussion/issues/1650

Connected to #1547

**Comment by JosephMcArthur** [comment]
expected to be closed by #1704

**Comment by markmacgillivray** [comment]
Closing this one as backups are covered in the new system now.

**Comment by JosephMcArthur** [comment]
Just to be clear, does that mean we've done the work to have the s3 backups, or, there is another issue.

Joseph McArthur
Assistant Director: Right to Research Coalition ( http://righttoresearch.org )
Director / Founder: Open Access Button ( http://openaccessbutton.org )
Twitter: @Mcarthur_Joe ( https://twitter.com/Mcarthur_Joe )
Skype: joseph_mcarthur

**Comment by markmacgillivray** [comment]
s3 backups has not been done, we now have multiple layers of data across cloudflare kv and our backend index. We will be doing more work to make use of improved s3 capabilities, but those will probably only be used for storing files rather than backing up our indexes, and there is an issue for that. We could enable s3 index backups as well once that issue is complete, but so far whilst reviewing along with building the new system I've not come across any indexes that I feel really need it so much any more, because those are the ones we'll now have in KV and also in an index. So rather than doing the work of turning on s3 backups for the index, I've been doing the work of spreading data (and copies) over different areas of the new system, and improving the fallback nature of the code that we use to query indexes of things like crossref, so that even if we lost our own copy we would just start querying from them again and rebuilding our local copy without our system going down (it would just become a bit slower for a while when looking up new or less popular articles).

**Comment by JosephMcArthur** [comment]
okay great thanks! If you think that's safe then I think it's safe :)

Joseph McArthur
Assistant Director: Right to Research Coalition ( http://righttoresearch.org )
Director / Founder: Open Access Button ( http://openaccessbutton.org )
Twitter: @Mcarthur_Joe ( https://twitter.com/Mcarthur_Joe )
Skype: joseph_mcarthur
